### DISCUSSION TITLE
Security Alerts / Security Dashboard

### DISCUSSION THREAD

**Discussion by psteinroe** [supabase/supabase#9228] [Feature Requests]
Hey,

for most developers I think the biggest pitfalls when working with supabase are around security aspects. Beside a better documentation, a lot of new users would probably benefit from a security dashboard, which would e.g. list security definer functions in public schemas, tables without rls enabled, and similar low hanging fruits.

**Comment by thorwebdev** [comment]
This is a great idea, thanks! I've flagged with the team and will have them follow up.
